ReleaseNotes.md

```diff
@@ -8,6 +8,32 @@ information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeo
 additional questions or comments.
 
 ## Release History
+### 04/07/2023 -
+##### Version microsoft/oms:3.1.6 Version mcr.microsoft.com/azuremonitor/containerinsights/ciprod:3.1.6 (linux)
+##### Version microsoft/oms:win-3.1.6 Version mcr.microsoft.com/azuremonitor/containerinsights/ciprod:win-3.1.6 (windows)
+##### Code change log
+- Common
+  - Update prometheus.io/scheme default description in configmap
```

Comment:
schema

Reply:
the wording should be right since it is coming from configmap